modules/infinite-scroll/infinity.php

```diff
@@ -1272,10 +1271,6 @@ function query() {
 			$results['type'] = 'empty';
 		}
 
-		if ( is_customize_preview() ) {
```

Comment:
Thanks for your review. I have added a check to ensure that it will work in 4.6 and also a comment to delete this once 4.8 is released.